packages/grid/_modules_/grid/utils/utils.ts

```diff
@@ -31,3 +31,6 @@ export function localStorageAvailable() {
 export function escapeRegExp(value: string): string {
   return value.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, '\\$&');
 }
+
+export const setInInterval = (value: number, min: number, max: number) =>
```

Comment:
I find `set` prefix confusing because it is not a setter. `get`, or `transform` could make sense. NumPy calls this function [clip](https://numpy.org/doc/stable/reference/generated/numpy.clip.html)

Reply:
A more commonly used name: `clamp`.

https://github.com/mui-org/material-ui/blob/eb01f34ed9b680f4b893cacd73b846c2284a429a/packages/mui-system/src/colorManipulator.js#L10

https://github.com/mui-org/material-ui/blob/eb01f34ed9b680f4b893cacd73b846c2284a429a/packages/mui-core/src/SliderUnstyled/SliderUnstyled.js#L25